Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGridColumnHeader.cs

```diff
@@ -712,7 +712,6 @@ private void DataGridColumnHeader_PointerReleased(object sender, PointerRoutedEv
                     {
                         // Completed a click or tap without dragging, so raise the DataGrid.Sorting event.
                         InvokeProcessSort();
-                        handled = true;
```

Comment:
Did this mean to set e.Handled to true instead? Is this just a hidden bug? @RBrid

Reply:
@vgromfeld `e.Handled` is set on [L757](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/c998f0985df9a229066267877aa0d69633afd22c/Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGridColumnHeader.cs#L757), so this shouldn't be removed? Is this a Roslyn bug?

Reply:
Re-reading the code, we have another `handled = true;` at line 754 at the end of the parent `if` block. This means that `e.Handled` at line 757 will properly be set to true when going into the `if` branch containing this `case` statement.

We can safely remove this unneeded affectation

---
In reply to: [391968965](https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3155#discussion_r391968965) [](ancestors = 391968965)

Reply:
Got it, good catch! :)